styles/ldf-client.css

```diff
@@ -98,7 +98,7 @@ fieldset {
   margin: 0;
 }
 .ldf-client, .ldf-client * {
-  max-width: 100%;
+ /* max-width: 100%;*/
```

Comment:
Not sure if we want this. Can you try just removing `.ldf-client *` instead?

Reply:
Done, removing `.ldf-client *` helped